tests/components/sonos/test_init.py

```diff
@@ -84,7 +99,9 @@ async def test_async_poll_manual_hosts_warnings(
     manager.hosts.add("10.10.10.10")
     with caplog.at_level(logging.DEBUG), patch.object(
         manager, "_async_handle_discovery_message"
-    ), patch("homeassistant.components.sonos.async_call_later"), patch(
+    ), patch(
+        "homeassistant.components.sonos.async_call_later"
```

Comment:
We probably shouldn’t be patching async_call_later here but that’s an existing issue

Reply:
Agreed, that's from a prior test.  I'll get that fixed in a new pull request.